third_party/move/move-compiler-v2/tests/ability-check/ability_violation.exp

```diff
@@ -0,0 +1,75 @@
+
+Diagnostics:
+error: type `ability::Impotent` does not have the `key` ability
+   ┌─ tests/ability-check/ability_violation.move:4:3
+   │
+ 4 │         move_from<T>(addr);
+   │         ^^^^^^^^^^^^^^^^^^ required because of storage operation here
+   ·
+14 │         move_from_no_key<Impotent>(addr);
+   │         -------------------------------- from a call inlined at this callsite
+
+error: value of type `ability::Impotent` does not have the `drop` ability
+   ┌─ tests/ability-check/ability_violation.move:4:3
+   │
+ 4 │         move_from<T>(addr);
+   │         ^^^^^^^^^^^^^^^^^^ implicitly dropped here since not longer used
```

Comment:
Sorry to keep nitpicking, but "since not longer used" is painful.  "since it is no longer used" (best) or (at least) "since no longer used", please.

Reply:
Done